src/Microsoft.TemplateEngine.TemplateLocalizer/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Generally we do not include launchSettings.json - is it intentional?

Reply:
Nope. I will remove this.

Reply:
We want to add it to .gitignore

Reply:
Added to .gitignore file.